index.html

```diff
@@ -1,417 +1,245 @@
+        <li class="list-item-icon"><a href="#" class="link link-icon fab fa-medium-m"></a></li>
+    </ul>
+    </div>
+    </section>
+    <section class="page-section works-section" id="portfolio-section">
+        <div class="box-join-title">
+            <h2 class="title section-title title-l1">My Recent Works</h2>
+        <hr class="h-rule">
+        </div>
+          <div class="works-container">
+            <div class="posts-display">
+              <figure class="post-image">
+                <img src="images/img-placeholder-display.png" alt="Placeholder image">
+              </figure>
+              <div class="post-information">
+                <p class="post-title">Multi-Post Stories</p>
+                <p class="post-description description-text"> A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard</p>
+                <ul class="post-tags ul-list horizontal-list">
+                  <li class="list-item-text">Html</li>
+                  <li class="list-item-text">Css</li>
+                  <li class="list-item-text">Bootstrap</li>
+                  <li class="list-item-text">Ruby</li>
+                </ul>
+                <button type="button" name="button" class="enable-button post-button">See project</button>
+              </div>
             </div>
-        </section>
-        <section class="about" id="about-section">
-            <div class="about-details">
-                <div class="abt-container">
-
-
-                    <div class="abt">
-
-                        <div class="d-abt">
-
-                            <div class="abt-contain">
-
-
-                                <h2>
-                                    About me
-                                </h2>
-
-                                <div class="abt-details">
-                                    <p>
-                                        Hello I’m a software developer! I can help you build a product , feature or
-                                        website
-                                        Look through some of my work and experience! If you like what you see and have a
-                                        project you need coded, don’t hestiate to contact me.
-
-                                    </p>
-                                </div>
-                                <div class="abt-btn">
-                                    <button class="get-btn">Get My Resume</button>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <hr class="bg">
-                </div>
+            <div class="posts-list grid-container">
+              <article class="post-article grid-item active-article">
+                <figure class="post-image post-image-desktop">
```

Comment:
[OPTIONAL] A tip for you: I've noticed that you made the first card different from the others, just like in the design:
Your design:
![image](https://user-images.githubusercontent.com/42799579/193392147-aedba2e0-bc86-4216-9794-00fabc1effc8.png)

If you pay closer attention, you will see that in the design there is a cursor hovering over this first card:
Figma design:
![image](https://user-images.githubusercontent.com/42799579/193004940-9a5ae8e6-3116-429e-843c-b470fb4c26d3.png)
This means that this card isn't different from the other ones, the design is showing to you the _hover_ effect of the cards. 😉
In short, all your cards should have the title/description but, when hovered, the project picture appears clearly and the _See Project_ button is displayed.
Keep up the good work!